Source/ACE.Server/Factories/LootGenerationFactory.cs

```diff
@@ -2317,6 +2338,42 @@ public static int CreateFood()
             return foodType;
         }
 
+        public static WorldObject CreateRandomScroll(int tier)
+        {
+            uint weenieID;
+            WorldDatabase wDb = new WorldDatabase();
```

Comment:
You can reference the world database via DatabaseManager.

We shouldn't be instantiating the databases in any other place, other than test projects.